operator/pkg/ciliumidentity/pod.go

```diff
@@ -3,8 +3,114 @@
 
 package ciliumidentity
 
-import "github.com/cilium/cilium/pkg/k8s/resource"
+import (
+	"context"
+	"time"
+
+	"k8s.io/client-go/util/workqueue"
+
+	"github.com/cilium/cilium/pkg/k8s/resource"
+	slim_core_v1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
 
 func podResourceKey(podName, podNamespace string) resource.Key {
 	return resource.Key{Name: podName, Namespace: podNamespace}
 }
+
+func (c *Controller) processPodEvents(ctx context.Context) error {
+	for event := range c.pod.Events(ctx) {
+		switch event.Kind {
+		case resource.Upsert:
+			c.logger.Debug("Got Upsert Pod event",
+				logfields.K8sPodName, event.Key.String())
+
+			c.onPodUpdate(event.Object)
+		case resource.Delete:
+			c.logger.Debug("Got Delete Pod event",
+				logfields.K8sPodName, event.Key.String())
+
+			c.onPodUpdate(event.Object)
+		}
+		event.Done(nil)
+	}
+	return nil
+}
+
+// onPodUpdate pushes a CID create to the CID work queue if there is no matching
+// CID for the security labels.
+func (c *Controller) onPodUpdate(pod *slim_core_v1.Pod) {
+	c.enqueuePodReconciliation(podResourceKey(pod.Name, pod.Namespace), 0)
+}
+
+func (c *Controller) initPodQueue() {
+	c.logger.Info("CID controller work queue configuration for Pod",
+		logfields.WorkQueueSyncBackOff, defaultSyncBackOff)
```

Comment:
If you are going to log the configuration, you should log all of the values:

* defaultSyncBackOff
* maxSyncBackOff